vyxal/elements.py

```diff
@@ -1478,18 +1478,25 @@ def index_indices_or_cycle(lhs, rhs, ctx):
     (any, fun) -> Repeatedly apply b to a until cycle is formed, then
```

Comment:
Llooks like `İ` has retained the same behavior despite the change in docs, but if you're updating elements.yaml, it'd be good to change this too, for consistency.